Public/Stop-SeDriver.ps1

```diff
@@ -7,12 +7,21 @@ function Stop-SeDriver {
         [OpenQA.Selenium.IWebDriver]
         $Target = $Global:SeDriver
     )
-
+    $TextInfo = (Get-Culture).TextInfo
     if (($null -ne $Target) -and ($Target -is [OpenQA.Selenium.IWebDriver])) {
```

Comment:
Very late, but - this if statement isn't needed; parameter validation is already verifying all of these things anyway. Might be a candidate to trim out in a future cleanup. 🙂

Reply:
@vexx32 

Good point for the type validation.

The `if` is still partially needed though as `-Target` is an optional parameter that will inherit from `$Global:SeDriver` if unspecified. 

While the parameter validation will take care of rejecting `$Global:SeDriver` if it contains a wrong type, it goes right through with a null value, hence making the null validation needed (but not the type). 

This example is why the `$null` validation (but not type) is needed.
```
# WARNING: A valid <IWebDriver> must be provided.
# That optional parameter assignment do not bypass the type, but did bypass the AllowNull requirement
$Global:SeDriver = $null
Stop-SeDriver
```

Based on the fact that Powershell does reject `$Global:SeDriver` when it has the wrong type, I am thinking that it is a bug with parameter validation (or for sure a weird quirk of the engine) 


Testing all other scenarios
```

#Error : The argument is null or empty.
Stop-SeDriver -Target $null  

#Cannot process argument transformation on parameter 'Target'
Stop-SeDriver -Target 'IAmASneakyStringShhhht'


#Cannot process argument transformation on parameter 'Target'
$Global:SeDriver = 'SneakyStringAttempt#2'
Stop-SeDriver

# WARNING: A valid <IWebDriver> must be provided.
# That optional parameter assignment do not bypass the type, but did bypass the AllowNull requirement
$Global:SeDriver = $null
Stop-SeDriver
```

Reply:
It will be cleaned up as part of the rewrite related to the `-AsDefaultDriver` / `Get-SeDriver` (#93) and Start-SeDriver (#101) enhancement.

Reply:
Good point. I was forgetting that validation attributes don't apply to the default value of a parameter, sorry!

🤔 wonder if we're better of checking `$PSBoundParameters.ContainsKey('Target')` as part of the internal check, not sure. As ever, use what works. 😛